frontend/apps/app/components/BuildPage/BuildPage.tsx

```diff
@@ -70,7 +71,14 @@ export async function BuildPage({ projectId, branchOrCommit }: Props) {
     throw new Error('Schema could not be parsed')
   }
 
+  const adaptedSchema = adaptSchemaForChatbot(schema)
```

Comment:
IMO: I was reading this implementation and it did not seem necessary to convert the Schema structure. Was there a reason for this one? cc @FunamaYukina 

https://github.com/liam-hq/liam/blob/865c8a725e3dc22f66874ed8f6bbe8778e9d9a16/frontend/apps/app/features/schemas/components/Chatbot/utils/schemaAdapter.ts

Reply:
Thank you for noticing! I will check and try to remove it if it is not needed!🙏